test/functional/apps/discover/group6/_sidebar.ts

```diff
@@ -28,10 +28,27 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const filterBar = getService('filterBar');
   const fieldEditor = getService('fieldEditor');
   const dataViews = getService('dataViews');
+  const queryBar = getService('queryBar');
   const retry = getService('retry');
   const dataGrid = getService('dataGrid');
+  const log = getService('log');
   const INITIAL_FIELD_LIST_SUMMARY = '48 available fields. 5 empty fields. 4 meta fields.';
 
+  const expectFieldListDescription = async (expectedNumber: string) => {
+    return await retry.try(async () => {
+      await discover.waitUntilSearchingHasFinished();
+      await unifiedFieldList.waitUntilSidebarHasLoaded();
+      const ariaDescription = await unifiedFieldList.getSidebarAriaDescription();
+      if (ariaDescription !== expectedNumber) {
+        log.warning(
+          `Expected Sidebar Aria Description: ${expectedNumber}, got: ${ariaDescription}`
+        );
+        await queryBar.submitQuery();
```

Comment:
that's the code that should help to catch an occacional flakiness, resubmitting the query, so available fields should be returned like expected